spk/squidguard/src/etc/squidclamav.conf.tpl

```diff
@@ -14,7 +14,7 @@
 maxsize 5000000
 
```

Comment:
Might as well keep this a `.conf` file, you rename it immediately during installation. What's the point then? ;)